azurerm/data_source_api_management_test.go

```diff
@@ -26,6 +26,7 @@ func TestAccDataSourceAzureRMApiManagement_basic(t *testing.T) {
 					resource.TestCheckResourceAttr(dataSourceName, "sku.0.capacity", "1"),
 					resource.TestCheckResourceAttr(dataSourceName, "sku.0.name", "Developer"),
 					resource.TestCheckResourceAttr(dataSourceName, "tags.%", "0"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "public_ip_addresses"),
```

Comment:
can we ensure there's items in this list? we can do that via:

```suggestion
					resource.TestCheckResourceAttrSet(dataSourceName, "public_ip_addresses.#"),
```

Reply:
Improved the test. Thanks for the suggestion.